src/components/Home.css

```diff
@@ -0,0 +1,97 @@
+.home {
+  position: relative;
+}
+
+.headline {
+  color: var(--color-two);
+  margin: 20px 0;
+  font-size: 2.5rem;
+  padding-left: 10px;
+  padding-bottom: 5px;
+  width: 185px;
+  background-color: rgba(204, 168, 169, 0.116);
+  background-blend-mode: hard-light;
+  backdrop-filter: blur(5px);
+}
+
+.headline-wrapper {
+  width: 185px;
+  height: 28px;
+  position: relative;
+  background-color: rgba(204, 168, 169, 0.376);
```

Comment:
ahem custom variables ahem

Reply:
i tried using the custum variable, but it doesn't work if i want to add opacity